### DISCUSSION TITLE
run a query by pointing to the query script file

### DISCUSSION THREAD

**Discussion by computechrr** [mithrandyr/SimplySql#149] [General]
I have a query that is a nightmare trying to create as a string in powershell due to all the special charatcers. it loads a data file flushes tables etc.  is there a way i can point to the existing query file to run it instead (mariadb)?
currently i use the mariadb command line in the script to run the query by pointing to the existing query file.

**Comment by mithrandyr** [comment]
@computechrr  -- yeah, you can do this, but you will have to use other cmdlets within PowerShell to handle reading from the file and then pass that information to the SimplySql commands.

```powershell
# use a variable
$query = Get-Content -Path .\example.sql
Invoke-SqlQuery -Query $query

# get the file content inline
Invoke-SqlQuery -Query (Get-Content -Path .\example.sql)
```

**Comment by computechrr** [comment]
did not think about that

thank you very much

Ted
________________________________
